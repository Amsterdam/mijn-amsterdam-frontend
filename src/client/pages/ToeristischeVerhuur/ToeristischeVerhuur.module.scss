@import '../../styles/_global.scss';

.ToeristischeVerhuur {
  @include select(PageContent) {
    padding-bottom: $page-content-bottom-space;
  }
}

.NoBorder {
  border-top: none;
}

.Detail {
  padding-top: 1.6rem;
}

.DetailText {
  padding-bottom: 0.4rem;
  display: block;
  margin: 0;
  font-weight: normal;
  color: $color-neutral-grey4;
}

.InvertedLabel {
  color: $color-black;
  display: block;
  font-weight: 500;
  font-size: 1.8rem;
  margin: 0;
  padding-right: 16rem;

  @include mq-phone {
    padding-right: 0;
  }
}

.TableTop {
  margin-top: 2.6rem;
  width: 100%;
  border: 0;
  border-bottom: 0;
  border-collapse: collapse;
}

.RegistrationNumber {
  margin-bottom: 2.6rem;
}

.SectionNoBorderBottom {
  border: 0;
}

.SectionBorderTop {
  border-top: 0.1rem $color-neutral-grey2 solid;
  padding-top: 2rem;
}

.Table {
  tr td {
    width: 25%;
  }
}
