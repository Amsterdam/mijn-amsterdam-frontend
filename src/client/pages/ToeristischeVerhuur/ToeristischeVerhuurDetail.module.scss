@import '../../styles/_global.scss';

.VanTot {
  &_Col1 {
    grid-column-start: 1;
    grid-column-end: span 4;
  }

  &_Col2 {
    grid-column-start: 5;
    grid-column-end: span 4;
  }
}

.Address {
  white-space: pre-line;
  display: block;
}
