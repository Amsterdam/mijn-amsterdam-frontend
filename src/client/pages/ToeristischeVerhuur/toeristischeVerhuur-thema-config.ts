import cloneDeep from 'lodash.clonedeep';
import { generatePath } from 'react-router-dom';

import {
  LVVRegistratie,
  ToeristischeVerhuurVergunning,
} from '../../../server/services/toeristische-verhuur/toeristische-verhuur-config-and-types';
import { AppRoutes } from '../../../universal/config/routes';
import { entries } from '../../../universal/helpers/utils';
import {
  DisplayProps,
  WithDetailLinkComponent,
} from '../../components/Table/TableV2';
import {
  ListPageParamKind as ListPageParamKindVergunningen,
  listPageParamKind as listPageParamKindVergunningen,
  tableConfig as tableConfigVergunningen,
} from '../Vergunningen/Vergunningen-thema-config';

const DISPLAY_PROPS_HUIDIGE_VERGUNNINGEN: DisplayProps<
  WithDetailLinkComponent<ToeristischeVerhuurVergunning>
> = {
  detailLinkComponent: 'Zaaknummer',
  title: 'Soort vergunning',
  displayStatus: 'Status',
  dateStartFormatted: 'Vanaf',
  dateEndFormatted: 'Tot',
};

const DISPLAY_PROPS_LVV_REGISTRATIES: DisplayProps<LVVRegistratie> = {
  registrationNumber: 'Registratienummer',
  address: 'Adres',
  agreementDateFormatted: 'Geregistreerd op',
};

export type ListPageParamKind = ListPageParamKindVergunningen;

export const listPageTitle = {
  [listPageParamKindVergunningen.inProgress]: 'Lopende aanvragen',
  [listPageParamKindVergunningen.actual]: 'Huidige vergunningen',
  [listPageParamKindVergunningen.historic]: 'Eerdere en afgewezen vergunningen',
} as const;

export const routes = {
  listPage: AppRoutes['TOERISTISCHE_VERHUUR/VERGUNNING/LIST'],
  detailPageVergunning: AppRoutes['TOERISTISCHE_VERHUUR/VERGUNNING'],
  themaPage: AppRoutes.TOERISTISCHE_VERHUUR,
} as const;

export const tableConfig = Object.fromEntries(
  entries(cloneDeep(tableConfigVergunningen)).map(([kind, tableConfig]) => {
    return [
      kind,
      {
        ...tableConfig,
        displayProps:
          kind === listPageParamKindVergunningen.actual
            ? DISPLAY_PROPS_HUIDIGE_VERGUNNINGEN
            : tableConfig.displayProps,
        title: listPageTitle[kind],
        filter: (vergunning: ToeristischeVerhuurVergunning) =>
          tableConfig.filter(vergunning),
        listPageRoute: generatePath(routes.listPage, {
          kind,
        }),
      },
    ];
  })
);

export const tableConfigLVVRegistraties = {
  title: 'Registratienummer(s) toeristische verhuur',
  displayProps: DISPLAY_PROPS_LVV_REGISTRATIES,
} as const;

export const THEMA_DETAIL_TITLE_DEFAULT = 'Vergunning toeristische verhuur';
