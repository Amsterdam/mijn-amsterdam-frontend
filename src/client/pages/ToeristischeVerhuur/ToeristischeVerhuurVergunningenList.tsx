import { useParams } from 'react-router';

import { ListPageParamKind } from './toeristischeVerhuur-thema-config';
import { useToeristischeVerhuurThemaData } from './useToeristischeVerhuur.hook';
import { ListPagePaginated } from '../../components/ListPagePaginated/ListPagePaginated';

export function ToeristischeVerhuurVergunningen() {
  const { kind } = useParams<{ kind: ListPageParamKind }>();
  const {
    vergunningen,
    tableConfigVergunningen,
    routes,
    isLoading,
    isError,
    breadcrumbs,
  } = useToeristischeVerhuurThemaData();
  const listPageTableConfig = tableConfigVergunningen[kind];

  return (
    <ListPagePaginated
      items={vergunningen
        .filter(listPageTableConfig.filter)
        .sort(listPageTableConfig.sort)}
      tableClassName={listPageTableConfig.className}
      title={listPageTableConfig.title}
      appRoute={routes.listPage}
      appRouteParams={{ kind }}
      breadcrumbs={breadcrumbs}
      displayProps={listPageTableConfig.displayProps}
      isLoading={isLoading}
      isError={isError}
    />
  );
}
