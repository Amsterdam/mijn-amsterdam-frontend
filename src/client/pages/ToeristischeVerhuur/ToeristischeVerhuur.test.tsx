import { render, screen } from '@testing-library/react';
import { generatePath } from 'react-router-dom';
import { MutableSnapshot } from 'recoil';
import toeristischeVerhuurRegistraties from '../../../server/mock-data/json/registraties-toeristische-verhuur.json';
import vergunningenData from '../../../server/mock-data/json/vergunningen.json';
import {
  transformVergunningenToVerhuur,
  VakantieverhuurVergunning,
} from '../../../server/services/toeristische-verhuur';
import {
  toeristischeVerhuurVergunningTypes,
  transformVergunningenData,
  Vergunning,
  VergunningenData,
} from '../../../server/services/vergunningen';
import { AppRoutes } from '../../../universal/config';
import { ApiSuccessResponse } from '../../../universal/helpers/api';
import { CaseType } from '../../../universal/types/vergunningen';
import { AppState } from '../../AppState';
import { appStateAtom } from '../../hooks/useAppState';
import MockApp from '../MockApp';
import ToeristischeVerhuur from './ToeristischeVerhuur';

const vergunningen = transformVergunningenData(
  vergunningenData as ApiSuccessResponse<VergunningenData>
)
  .filter((vergunning: Vergunning): vergunning is VakantieverhuurVergunning =>
    toeristischeVerhuurVergunningTypes.includes(vergunning.caseType)
  )
  .map((vergunning) => ({
    ...vergunning,
    link: { to: AppRoutes.TOERISTISCHE_VERHUUR, title: vergunning.title },
  }));

type VerhuurState = Pick<AppState, 'TOERISTISCHE_VERHUUR'>;

const testState: VerhuurState = {
  TOERISTISCHE_VERHUUR: {
    status: 'OK',
    content: {
      daysLeft: 26,
      registraties: toeristischeVerhuurRegistraties.content,
      title: 'Vakantieverhuur',
      vergunningen: transformVergunningenToVerhuur(vergunningen),
    },
  },
};

const testState2: VerhuurState = {
  TOERISTISCHE_VERHUUR: {
    status: 'OK',
    content: {
      daysLeft: 2,
      registraties: [],
      title: 'Bed & Breakfast',
      vergunningen: transformVergunningenToVerhuur(vergunningen),
    },
  },
};

const testState3: VerhuurState = {
  TOERISTISCHE_VERHUUR: {
    status: 'OK',
    content: {
      daysLeft: 2,
      registraties: toeristischeVerhuurRegistraties.content,
      title: 'Vakantieverhuur',
      vergunningen: transformVergunningenToVerhuur(vergunningen).filter(
        (vergunning) => vergunning.caseType === CaseType.BBVergunning
      ),
    },
  },
};

const testState4: VerhuurState = {
  TOERISTISCHE_VERHUUR: {
    status: 'OK',
    content: {
      daysLeft: 2,
      title: 'Vakantieverhuur',
      registraties: toeristischeVerhuurRegistraties.content,
      vergunningen: transformVergunningenToVerhuur(vergunningen).filter(
        (vergunning) => vergunning.caseType !== CaseType.BBVergunning
      ),
    },
  },
};

const testState5: VerhuurState = {
  TOERISTISCHE_VERHUUR: {
    status: 'OK',
    content: {
      daysLeft: 0,
      title: 'Vakantieverhuur',
      registraties: toeristischeVerhuurRegistraties.content,
      vergunningen: transformVergunningenToVerhuur(vergunningen).filter(
        (vergunning) => vergunning.caseType !== CaseType.BBVergunning
      ),
    },
  },
};

function initializeState(snapshot: MutableSnapshot, state: VerhuurState) {
  snapshot.set(appStateAtom as any, state);
}

describe('<ToeristischeVerhuur />', () => {
  const routeEntry = generatePath(AppRoutes.TOERISTISCHE_VERHUUR);
  const routePath = AppRoutes.TOERISTISCHE_VERHUUR;

  const Component = ({ state }: { state: VerhuurState }) => (
    <MockApp
      routeEntry={routeEntry}
      routePath={routePath}
      component={ToeristischeVerhuur}
      initializeState={(snap) => initializeState(snap, state)}
    />
  );

  it('Matches the Full Page snapshot', () => {
    const { asFragment } = render(<Component state={testState} />);
    expect(asFragment()).toMatchSnapshot();
  });

  it('Shows page for B&B and Vakantieverhuur permits', () => {
    render(<Component state={testState} />);
    expect(screen.getByText('Vakantieverhuur')).toBeInTheDocument();
    expect(
      screen.getByText('Meer over toeristenbelasting')
    ).toBeInTheDocument();
    expect(
      screen.queryAllByText('Registratienummer toeristische verhuur').length
    ).toBe(1);
    expect(screen.getByText('E7B8 B042 8A92 37E5 0363')).toBeInTheDocument();
    expect(
      screen.getByText('U hebt nog 26 nachten dat u uw woning mag verhuren.')
    ).toBeInTheDocument();
    expect(screen.getAllByText('Vergunning vakantieverhuur').length).toBe(3);
    expect(screen.getAllByText('Vergunning bed & breakfast').length).toBe(3);
    expect(screen.getByText('Geplande verhuur')).toBeInTheDocument();
    expect(screen.getByText('Geannuleerde verhuur')).toBeInTheDocument();
    expect(screen.getByText('Afgelopen verhuur')).toBeInTheDocument();

    expect(
      screen.findAllByText(
        'Het is niet toegestaan om op hetzelfde adres zowel aan vakantieverhuur als bed & breakfast te doen.'
      )
    ).not.toBe(null);
  });

  it('Shows alert for missing registration numbers', () => {
    render(<Component state={testState2} />);

    expect(
      screen.getByText(
        /U moet daarom ook een landelijk registratienummer voor toeristische verhuur aanvragen./
      )
    ).toBeInTheDocument();
  });

  it('Shows B&B page', () => {
    render(<Component state={testState3} />);

    expect(
      screen.getByText('Meer over toeristenbelasting')
    ).toBeInTheDocument();

    expect(
      screen.getByText('Meer informatie over bed & breakfast')
    ).toBeInTheDocument();

    expect(
      screen.queryByText('Vergunning vakantieverhuur')
    ).not.toBeInTheDocument();
    expect(screen.getAllByText('Vergunning bed & breakfast').length).toBe(3);
  });

  it('Shows Vakantieverhuur page', () => {
    render(<Component state={testState4} />);

    expect(
      screen.getByText('Meer over toeristenbelasting')
    ).toBeInTheDocument();

    expect(
      screen.getByText('Meer informatie over particuliere vakantieverhuur')
    ).toBeInTheDocument();

    expect(screen.getAllByText('Vergunning vakantieverhuur').length).toBe(3);
    expect(
      screen.queryByText('Vergunning bed & breakfast')
    ).not.toBeInTheDocument();
  });

  it('Shows Different text about nights rent left', () => {
    render(<Component state={testState5} />);

    expect(
      screen.getByText(/Uw woning is dit kalenderjaar al 30 nachten verhuurd./)
    ).toBeInTheDocument();
  });
});
