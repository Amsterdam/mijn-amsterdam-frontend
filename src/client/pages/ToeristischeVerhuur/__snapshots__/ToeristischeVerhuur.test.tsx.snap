// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ToeristischeVerhuur /> Matches the Full Page snapshot 1`] = `
<DocumentFragment>
  <main
    class="Page OverviewPage ToeristischeVerhuur"
  >
    <header
      class="PageHeading hasIcon hasBackLink"
    >
      <span
        aria-hidden="true"
        class="Icon"
      >
        <svg
          aria-label="Toeristische verhuur"
          class="ChapterIcon"
          fill="black"
        >
          toeristische-verhuur.svg
        </svg>
      </span>
      <div
        class="HeadingInner"
      >
        <a
          class="Button Button__plain Button__lean Linkd BackLink"
          href="/"
        >
          <svg
            aria-hidden="true"
            class="Icon Icon__left"
            fill="#000"
            height="14px"
            width="14px"
          >
            Chevron-Left.svg
          </svg>
          Home
        </a>
        <h2
          class="Heading Heading__large"
        >
          Toeristische verhuur
        </h2>
      </div>
    </header>
    <div
      class="PageContent"
    >
      <p>
        Hieronder vindt u een overzicht van uw aanvragen voor toeristische verhuur.
      </p>
      <p>
        <a
          class="Button Button__plain Button__lean Linkd"
          href="https://www.amsterdam.nl/wonen-leefomgeving/wonen/vakantieverhuur/"
          rel="external noopener noreferrer"
        >
          <svg
            aria-hidden="true"
            class="Icon Icon__left"
            fill="#000"
            height="14px"
            width="14px"
          >
            Chevron-Right.svg
          </svg>
          Meer informatie over particuliere vakantieverhuur
        </a>
        <a
          class="Button Button__plain Button__lean Linkd"
          href="https://www.amsterdam.nl/wonen-leefomgeving/wonen/bedandbreakfast/"
          rel="external noopener noreferrer"
        >
          <svg
            aria-hidden="true"
            class="Icon Icon__left"
            fill="#000"
            height="14px"
            width="14px"
          >
            Chevron-Right.svg
          </svg>
          Meer informatie over bed & breakfast
        </a>
      </p>
      <div
        class="Detail"
      >
        <h3
          class="Heading Heading__tiny InvertedLabel"
        >
          U heeft nog 26 nachten dat u uw woning mag verhuren.
        </h3>
        <p
          class="DetailText"
        >
          Het aantal resterende nachten is gebaseerd op uw meldingen voor ingepland en afgelopen verhuur. Dit is zonder eventuele meldingen die dit jaar door een mede-verhuurder of vorige bewoner zijn gedaan. Kijk voor meer informatie bij 
          <a
            class="Button Button__inline Button__lean Linkd LinkedInline"
            href="https://www.amsterdam.nl/wonen-leefomgeving/wonen/vakantieverhuur/melden"
            rel="external noopener noreferrer"
          >
            Vakantieverhuur melden
          </a>
          . Aan de informatie op deze pagina kunnen geen rechten worden ontleend.
        </p>
      </div>
    </div>
    <section
      class="SectionCollapsible SectionBorderTop"
    >
      <h3
        class="Heading Heading__mediumLarge Title TitleWithItems"
      >
        <button
          aria-expanded="true"
          class="TitleToggle"
        >
          <svg
            aria-hidden="true"
            class="CaretIcon"
          >
            Chevron-Right.svg
          </svg>
           Geplande verhuur
        </button>
      </h3>
      <div
        aria-hidden="false"
        class="Panel"
        style="height: 0px;"
      >
        <div
          class="PanelInner"
        >
          <table
            class="Table Table"
          >
            <thead>
              <tr
                class="TableRow"
              >
                <th
                  class="DisplayProp"
                >
                  Start verhuur
                </th>
                <th
                  class="DisplayProp"
                >
                  Einde verhuur
                </th>
                <th
                  class="DisplayProp"
                >
                  Ontvangen op
                </th>
                <th
                  class="DisplayProp"
                >
                  Aantal nachten
                </th>
              </tr>
            </thead>
            <tbody>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayPropTitle"
                >
                  <a
                    class="Button Button__plain Button__lean Linkd"
                    href="/toeristische-verhuur"
                  >
                    <svg
                      aria-hidden="true"
                      class="Icon Icon__left"
                      fill="#000"
                      height="14px"
                      width="14px"
                    >
                      Chevron-Right.svg
                    </svg>
                    10 juli 2021
                  </a>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Einde verhuur:
                  </span>
                  <span>
                    14 juli 2021
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Ontvangen op:
                  </span>
                  <span>
                    10 mei 2021
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Aantal nachten:
                  </span>
                  <span>
                    4
                  </span>
                </td>
              </tr>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayPropTitle"
                >
                  <a
                    class="Button Button__plain Button__lean Linkd"
                    href="/toeristische-verhuur"
                  >
                    <svg
                      aria-hidden="true"
                      class="Icon Icon__left"
                      fill="#000"
                      height="14px"
                      width="14px"
                    >
                      Chevron-Right.svg
                    </svg>
                    10 juli 2029
                  </a>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Einde verhuur:
                  </span>
                  <span>
                    14 juli 2029
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Ontvangen op:
                  </span>
                  <span>
                    10 mei 2021
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Aantal nachten:
                  </span>
                  <span>
                    4
                  </span>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </section>
    <section
      class="SectionCollapsible"
    >
      <h3
        class="Heading Heading__mediumLarge Title TitleWithItems"
      >
        <button
          aria-expanded="false"
          class="TitleToggle"
        >
          <svg
            aria-hidden="true"
            class="CaretIcon"
          >
            Chevron-Right.svg
          </svg>
           Geannuleerde verhuur
        </button>
      </h3>
      <div
        aria-hidden="true"
        class="Panel"
        style="height: 0px;"
      >
        <div
          class="PanelInner"
        >
          <table
            class="Table Table"
          >
            <thead>
              <tr
                class="TableRow"
              >
                <th
                  class="DisplayProp"
                >
                  Start verhuur
                </th>
                <th
                  class="DisplayProp"
                >
                  Einde verhuur
                </th>
                <th
                  class="DisplayProp"
                >
                  Ontvangen op
                </th>
                <th
                  class="DisplayProp"
                >
                  Aantal nachten
                </th>
              </tr>
            </thead>
            <tbody>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayPropTitle"
                >
                  <a
                    class="Button Button__plain Button__lean Linkd"
                    href="/toeristische-verhuur"
                  >
                    <svg
                      aria-hidden="true"
                      class="Icon Icon__left"
                      fill="#000"
                      height="14px"
                      width="14px"
                    >
                      Chevron-Right.svg
                    </svg>
                    09 juni 2021
                  </a>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Einde verhuur:
                  </span>
                  <span>
                    12 juni 2021
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Ontvangen op:
                  </span>
                  <span>
                    22 mei 2021
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Aantal nachten:
                  </span>
                  <span>
                    3
                  </span>
                </td>
              </tr>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayPropTitle"
                >
                  <a
                    class="Button Button__plain Button__lean Linkd"
                    href="/toeristische-verhuur"
                  >
                    <svg
                      aria-hidden="true"
                      class="Icon Icon__left"
                      fill="#000"
                      height="14px"
                      width="14px"
                    >
                      Chevron-Right.svg
                    </svg>
                    05 juni 2021
                  </a>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Einde verhuur:
                  </span>
                  <span>
                    10 juni 2021
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Ontvangen op:
                  </span>
                  <span>
                    10 mei 2021
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Aantal nachten:
                  </span>
                  <span>
                    5
                  </span>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </section>
    <section
      class="SectionCollapsible"
    >
      <h3
        class="Heading Heading__mediumLarge Title TitleWithItems"
      >
        <button
          aria-expanded="false"
          class="TitleToggle"
        >
          <svg
            aria-hidden="true"
            class="CaretIcon"
          >
            Chevron-Right.svg
          </svg>
           Afgelopen verhuur
        </button>
      </h3>
      <div
        aria-hidden="true"
        class="Panel"
        style="height: 0px;"
      >
        <div
          class="PanelInner"
        >
          <table
            class="Table Table"
          >
            <thead>
              <tr
                class="TableRow"
              >
                <th
                  class="DisplayProp"
                >
                  Start verhuur
                </th>
                <th
                  class="DisplayProp"
                >
                  Einde verhuur
                </th>
                <th
                  class="DisplayProp"
                >
                  Ontvangen op
                </th>
                <th
                  class="DisplayProp"
                >
                  Aantal nachten
                </th>
              </tr>
            </thead>
            <tbody>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayPropTitle"
                >
                  <a
                    class="Button Button__plain Button__lean Linkd"
                    href="/toeristische-verhuur"
                  >
                    <svg
                      aria-hidden="true"
                      class="Icon Icon__left"
                      fill="#000"
                      height="14px"
                      width="14px"
                    >
                      Chevron-Right.svg
                    </svg>
                    02 juni 2021
                  </a>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Einde verhuur:
                  </span>
                  <span>
                    07 juni 2021
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Ontvangen op:
                  </span>
                  <span>
                    10 mei 2020
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Aantal nachten:
                  </span>
                  <span>
                    5
                  </span>
                </td>
              </tr>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayPropTitle"
                >
                  <a
                    class="Button Button__plain Button__lean Linkd"
                    href="/toeristische-verhuur"
                  >
                    <svg
                      aria-hidden="true"
                      class="Icon Icon__left"
                      fill="#000"
                      height="14px"
                      width="14px"
                    >
                      Chevron-Right.svg
                    </svg>
                    02 mei 2021
                  </a>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Einde verhuur:
                  </span>
                  <span>
                    03 mei 2021
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Ontvangen op:
                  </span>
                  <span>
                    23 april 2020
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Aantal nachten:
                  </span>
                  <span>
                    1
                  </span>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </section>
    <section
      class="SectionCollapsible SectionNoBorderBottom"
    >
      <h3
        class="Heading Heading__mediumLarge Title TitleWithItems"
      >
        <button
          aria-expanded="false"
          class="TitleToggle"
        >
          <svg
            aria-hidden="true"
            class="CaretIcon"
          >
            Chevron-Right.svg
          </svg>
           Vergunningen
        </button>
      </h3>
      <div
        aria-hidden="true"
        class="Panel"
        style="height: 0px;"
      >
        <div
          class="PanelInner"
        >
          <table
            class="Table TableVergunningen"
          >
            <thead>
              <tr
                class="TableRow"
              >
                <th
                  class="DisplayProp"
                >
                   
                </th>
                <th
                  class="DisplayProp"
                >
                  Status
                </th>
                <th
                  class="DisplayProp"
                >
                  Vanaf
                </th>
                <th
                  class="DisplayProp"
                >
                  Tot
                </th>
              </tr>
            </thead>
            <tbody>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayPropTitle"
                >
                  <a
                    class="Button Button__plain Button__lean Linkd"
                    href="/toeristische-verhuur"
                  >
                    <svg
                      aria-hidden="true"
                      class="Icon Icon__left"
                      fill="#000"
                      height="14px"
                      width="14px"
                    >
                      Chevron-Right.svg
                    </svg>
                    Vergunning bed & breakfast
                  </a>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Status:
                  </span>
                  <span>
                    Ontvangen
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Vanaf:
                  </span>
                  <span>
                     
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Tot:
                  </span>
                  <span>
                     
                  </span>
                </td>
              </tr>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayPropTitle"
                >
                  <a
                    class="Button Button__plain Button__lean Linkd"
                    href="/toeristische-verhuur"
                  >
                    <svg
                      aria-hidden="true"
                      class="Icon Icon__left"
                      fill="#000"
                      height="14px"
                      width="14px"
                    >
                      Chevron-Right.svg
                    </svg>
                    Vergunning bed & breakfast
                  </a>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Status:
                  </span>
                  <span>
                    In behandeling
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Vanaf:
                  </span>
                  <span>
                    22 oktober 2020
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Tot:
                  </span>
                  <span>
                    21 oktober 2021
                  </span>
                </td>
              </tr>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayPropTitle"
                >
                  <a
                    class="Button Button__plain Button__lean Linkd"
                    href="/toeristische-verhuur"
                  >
                    <svg
                      aria-hidden="true"
                      class="Icon Icon__left"
                      fill="#000"
                      height="14px"
                      width="14px"
                    >
                      Chevron-Right.svg
                    </svg>
                    Vergunning bed & breakfast
                  </a>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Status:
                  </span>
                  <span>
                    Verleend
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Vanaf:
                  </span>
                  <span>
                    22 oktober 2020
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Tot:
                  </span>
                  <span>
                    21 oktober 2021
                  </span>
                </td>
              </tr>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayPropTitle"
                >
                  <a
                    class="Button Button__plain Button__lean Linkd"
                    href="/toeristische-verhuur"
                  >
                    <svg
                      aria-hidden="true"
                      class="Icon Icon__left"
                      fill="#000"
                      height="14px"
                      width="14px"
                    >
                      Chevron-Right.svg
                    </svg>
                    Vergunning vakantieverhuur
                  </a>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Status:
                  </span>
                  <span>
                    Verleend
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Vanaf:
                  </span>
                  <span>
                    01 augustus 2020
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Tot:
                  </span>
                  <span>
                    30 september 2021
                  </span>
                </td>
              </tr>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayPropTitle"
                >
                  <a
                    class="Button Button__plain Button__lean Linkd"
                    href="/toeristische-verhuur"
                  >
                    <svg
                      aria-hidden="true"
                      class="Icon Icon__left"
                      fill="#000"
                      height="14px"
                      width="14px"
                    >
                      Chevron-Right.svg
                    </svg>
                    Vergunning vakantieverhuur
                  </a>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Status:
                  </span>
                  <span>
                    Ingetrokken
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Vanaf:
                  </span>
                  <span>
                    01 juni 2020
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Tot:
                  </span>
                  <span>
                    31 mei 2021
                  </span>
                </td>
              </tr>
              <tr
                class="TableRow"
              >
                <td
                  class="DisplayPropTitle"
                >
                  <a
                    class="Button Button__plain Button__lean Linkd"
                    href="/toeristische-verhuur"
                  >
                    <svg
                      aria-hidden="true"
                      class="Icon Icon__left"
                      fill="#000"
                      height="14px"
                      width="14px"
                    >
                      Chevron-Right.svg
                    </svg>
                    Vergunning vakantieverhuur
                  </a>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Status:
                  </span>
                  <span>
                    Verlopen
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Vanaf:
                  </span>
                  <span>
                    01 juni 2019
                  </span>
                </td>
                <td
                  class="DisplayProp"
                >
                  <span
                    class="DisplayPropLabel"
                  >
                    Tot:
                  </span>
                  <span>
                    31 mei 2020
                  </span>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </section>
    <div
      class="PageContent"
    >
      <div
        class="InfoDetail"
      >
        <h3
          class="Heading Heading__tiny Label"
        >
          Registratienummer toeristische verhuur
        </h3>
        <div
          class="Value"
        >
          <article
            class="RegistrationNumber"
          >
            <span>
              0363 E7B8 B042 8A92 37E5
            </span>
            <br />
            Schakelstraat 16  1014AW Amsterdam
          </article>
          <article
            class="RegistrationNumber"
          >
            <span>
              E7B8 B042 8A92 37E5 0363
            </span>
            <br />
            Amstel 1  1017AB Amsterdam
          </article>
        </div>
      </div>
    </div>
  </main>
</DocumentFragment>
`;
